src/yang/ietf-bgp.yang

```diff
@@ -689,23 +689,37 @@ module ietf-bgp {
             }
             container messages {
               description
-                "Counters for BGP messages sent and received from the
+		"Counters for BGP messages sent and received from the
                  neighbor";
-              leaf in-total-messages {
-                type yang:zero-based-counter32;
+              leaf messages-received {
```

Comment:
Since the container is called “messages”, no need to prefix the child leaf with the parent container name.

Reply:
I've renamed this from messages-received/sent to total-received/sent.